RFC-by-stage/1-approved/distribute-css-as-unprefixed.md

```diff
@@ -0,0 +1,30 @@
+---
+Status: `Proposal` # Please do not change this.
+Implementer: # It will be changed upon merging and as it moves through the RFC stages
+---
+
+# Distribute CSS unminified and unprefixed
+
+## The issue to be solved
+
+CSS is currently being distributed as minified and vendor-prefixed assets.
+
+This is unnecessary since users (developing apps using npm) process styles for minification and vendor-prefixing, therefore css "as modules" should ship without being processed. This issue was raised by @liamfiddler [here](https://github.com/designsystemau/RFCs/pull/11#discussion_r724769922).
+
+Where users want to consume full CSS however, they may still expect the asset to be vendor prefixed and minified. This scenario might be handled better by delivering styles from a CDN like [UNPKG](https://unpkg.com/).
```

Comment:
So perhaps we leave both unprefixed and prefixed css files in with dist folder for consumers to choose from?